### PR TITLE
Implement minified js system

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #198 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

Implement current minified js system that runs the js for examples on the demo files.

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by jordanjones243** [comment]
This PR is currently sitting in draft mode because the `focus` example is not working as intended.

**Comment by jordanjones243** [comment]
Take note: the programmatic focus example was removed from the repo. Adding focus programmatically does not give the same UX as when a user applys focus to a hyperlink via tabbing.

**Comment by jordanjones243** [comment]
> Take note: the programmatic focus example was removed from the repo. Adding focus programmatically does not give the same UX as when a user applys focus to a hyperlink via tabbing.

I can confirm that this example does not work on the localhost in `main`, nor on the [docsite](https://auro.alaskaair.com/components/auro/hyperlink#gsc.tab=0)

File: package.json [hunk 1]
```diff
@@ -136,11 +136,12 @@
     "web components"
   ],
   "scripts": {
-    "build": "npm-run-all build:sass sass:render scssLint dist:js bundler postinstall build:api build:docs",
+    "build": "npm-run-all build:sass sass:render build:api test build:docs bundler build:demoScripts postinstall types",
```
Comment (jason-capsule42) [inline package.json:139]: Shouldn't we be running the linters still? Without it here it never happens unless we do it manually.

Comment (jordanjones243) [reply package.json:139]: Adding that back in now

**Pull request merged**